modules/nodes-descheduler-configuring-strategies.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-strategies_{context}"]
+= Configuring descheduler strategies
+
+You can configure which strategies the descheduler uses to evict Pods.
+
+.Prerequisites
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
```

Comment:
I tried to follow the guide to install & configure descheduler strategies -> Once the install is done using the console, i do not see kubedeschedulers object cluster being available. 

[ramakasturinarra@dhcp35-60 ~]$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
Error from server (NotFound): kubedeschedulers.operator.openshift.io "cluster" not found
[ramakasturinarra@dhcp35-60 ~]$ oc get project | grep "descheduler"
openshift-kube-descheduler-operator                                    Active
[ramakasturinarra@dhcp35-60 ~]$ oc project openshift-kube-descheduler-operator
Now using project "openshift-kube-descheduler-operator" on server "https://api.knarra-441.qe.devcluster.openshift.com:6443".
[ramakasturinarra@dhcp35-60 ~]$ oc get kubedeschedulers.operator.openshift.io
No resources found in openshift-kube-descheduler-operator namespace.
[ramakasturinarra@dhcp35-60 ~]$ oc get kubedescheduler.operator.openshift.io
No resources found in openshift-kube-descheduler-operator namespace.

So before running this command user needs to click on the KubeDescheduler from the console itself and create the  object and after that the will be able to edit the object and configure strategies.

1) click on installed operators
2) click on descheduler operator
3) click on KubeDescheduler
4) click on the button "create KubeDescheduler"
5) click create
Above steps needs to be added before "Edit the KubeDescheduler" object for the "oc edit" step to work.

Reply:
Thanks @kasturinarra, I added steps for creating the instance after installing the operator. Can you please take another look? I've also addressed your other feedback.

Reply:
Changes added as part of instance creation after installing the operator looks good to me.